config.yaml

```diff
@@ -1,49 +1,62 @@
 options:
   path:
-      default: ""
       type: string
-      description: "Either an organization name or repository in 'org/repo' format"
-  token:
       default: ""
+      description: >
+        The path of the GitHub repo to register the self-hosted runners. Either an organization name
+        or repository in 'org/repo' format.
+  token:
       type: string
-      description: "Personal Access Token, required for auto registerting with the given url"
-  quantity:
-      default: 3
-      type: int
-      description: |
-        DEPRECATED: use `containers` and `virtual-machines`
-        Quantity of runners to maintain
+      default: ""
+      description:
+        The GitHub Personal Access Token for auto-registering the self-hosted runners.
   containers:
-      default: 0
       type: int
-      description: "Quantity of container runners to maintain"
-  virtual-machines:
       default: 0
+      description: >
+        The target quantity of container runners. This charm will spawn or destroy container
+        runners to match this setting.
+  virtual-machines:
       type: int
-      description: "Quantity of virtual machine runners to maintain"
+      default: 0
+      description: >
+        The target quantity of virtual machine runners. This charm will spawn or destroy virtual
+        machines runners to match this setting.
   vm-cpu:
-      default: 2
       type: int
-      description: "Number of CPUs for VMs."
+      default: 2
+      description: >
+        The number of CPUs used by each virtual machine runner. Does not affect container runners.
   vm-memory:
-    default: 7GiB
     type: string
-    description: "VM memory size."
+    default: 7GiB
+    description: >
+        The memory size used by each virtual machine runner. Does not affect container runners.
   vm-disk:
-    default: 10GiB
     type: string
-    description: "VM disk size."
+    default: 10GiB
+    description: >
+        The root disk size used by each virtual machine runner. Does not affect container runners.
   reconcile-interval:
-      default: 5
       type: int
-      description: "How frequently, in minutes, the runners should be reconciled"
+      default: 5
+      description: >
+        The number of minutes between each reconciliation between the current state of runners and
```

Comment:
"The number of minutes" -> "Minutes"
"between the current state of runners" -> "between the current state"

Reply:
fixed